setup.cfg

```diff
@@ -1,6 +1,6 @@
 [tool:pytest]
 testpaths = tests
-timeout = 5
+;timeout = 5
```

Comment:
good to keep something here, are some tests really taking more than 5 seconds?

Reply:
Oh, I overlooked it, I added it when I was working on the changes for v2, needed to run tests without timeout. Currently in my environment only `test_redis_sentinel_failure` fails because of timeout, and I think it should be expected, because with 5 retries and 1 second sleep `test_redis_sentinel_failure` will always run for at least 5 seconds. I think we need to fix this test (may be decrease max retries?).